src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area,Drawable {
```

Comment:
Don't begin class or method implementation with an empty line. The empty line after the class declaration on line 2 should be removed.